doc/sphinx/source/community/release_strategy.rst

```diff
@@ -190,6 +188,136 @@ Release branch
 ~~~~~~~~~~~~~~
 The release branch can be used to do some additional testing before the release, while normal development work continues in the master branch. It will be branched off from the master branch after the feature freeze and will be used to make the release on the release date. The only way to still get something included in the release after the feature freeze is to ask the release manager to cherry-pick a commit from the master branch into this branch.
 
+
+.. _How to make a release:
+
+How to make an ESMValTool release
+---------------------------------
+
+The release manager makes the release, assisted by the release manager of the
+previous release, or if that person is not available, another previous release
+manager. Perform the steps listed below with two persons, to reduce the risk of
+error.
+
+To make a new release of the package, follow these steps:
+
+1. Check the tests on GitHub Actions and CircleCI
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Check the ``nightly``
+`build on CircleCI <https://circleci.com/gh/ESMValGroup/ESMValTool/tree/master>`__
+and the
+`GitHub Actions run <https://github.com/ESMValGroup/ESMValTool/actions>`__.
+All tests should pass before making a release (branch).
+
+2. Increase the version number
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The version number is stored in ``esmvaltool/__init__.py``,
+``package/meta.yaml``, ``CITATION.cff``. Make sure to update all files.
+Also update the release date in ``CITATION.cff``.
+See https://semver.org for more information on choosing a version number.
+Make a pull request and get it merged into ``master``.
+
+3. Add release notes
+~~~~~~~~~~~~~~~~~~~~
+Use the script :ref:`draft_release_notes.py` to create create a draft of the
+release notes.
+This script uses the titles and labels of merged pull requests since the
+previous release.
+Review the results, and if anything needs changing, change it on GitHub and
+re-run the script until the changelog looks acceptable.
+Copy the result to the file ``doc/sphinx/source/changelog.rst``.
+Make a pull request and get it merged into ``master``.
+
+4. Create a release branch
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+Create a branch off the ``master`` branch and push it to GitHub.
+Ask someone with administrative permissions to set up branch protection rules
+for it so only you and the person helping you with the release can push to it.
+Announce the name of the branch in an issue and ask the members of the
+`ESMValTool development team <https://github.com/orgs/ESMValGroup/teams/esmvaltool-developmentteam>`__
+to run their favourite recipe using this branch.
+
+5. Cherry pick bugfixes into the release branch
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+If a bug is found and fixed (i.e. pull request merged into the
+``master`` branch) during the period of testing, use the command
+``git cherry-pick`` to include the commit for this bugfix into
```

Comment:
`git cherry-pick COMMIT_HASH` where `COMMIT_HASH` is the commit hash of the commit that needs to be cherry-picked

Reply:
Done!